src/finance_quote/modules/fixerio.py

```diff
@@ -0,0 +1,185 @@
+"""
+Fixer.io currency rates 
+Currently only implements the free public API access with yesterday's rates.
+"""
+import os
+import tempfile
+from datetime import datetime
+import logging
+from decimal import Decimal
+
+from finance_quote.base import Source, Quote
+import finance_quote
+
+try: import simplejson as json
+except ImportError: import json
+
+
+class FixerioQuote(Quote):
+    """ Fixer.io-specific quote """
+    pass
+
+
+class Fixerio(Source):
```

Comment:
should better be named FixerioSource for consistency

Reply:
Consistency with what? :)
The model I was trying to replicate was the original F::Q. It only has a root object and a bunch of module scripts.

Reply:
With the two other scrapers already in the lib (yahoo.py and quandl.py) and to be pythonic.
I do not think we should follow the structure/API of the perl package: it is old & it is perl. Let us inspire more from similar python packages (look on PyPI for quote, yahoo, ... for examples)

Ultimately, we could provide a CLI that is compatible with the f::q package so that it could be a drop-in replacement but it is not the first goal I think.